src/main/java/Calculator.java

```diff
@@ -0,0 +1,20 @@
+public class Calculator {
+
+    public double add(double a1, double a2){
+        return a1 + a2;
+    }
+
+    public double subtract(double a1, double a2){
```

Comment:
double is ok, since no requirements were for it.

But you can make it a little bit simpler, just use int every where.
Only in divide:
`public double divide(int a, int b)`